Angela/tenant content strategy update (#3388)
cloudflare/cloudflare-docs: products/tenant/src/content/tutorial/enabling-services.md

# Step 3: Enabling services

> In order for your customers to get the most out of Cloudflare, it is important that you can easily provision paid services so that customer's services are as fast and secure as possible. To do that, we are going to create some subscriptions for our customer, but first the customer needs a zone.

Reviewer: ```suggestion
In order for your customers to get the most out of Cloudflare, it is important that you can easily provision paid services so that your customer's services are as fast and secure as possible. To do that, we are going to create some subscriptions for our customer, but first the customer needs a zone.
```

## Creating a zone

> Creating a zone is similar to creating an API Client, but be sure to specify the customer's account when creating the zone.

Reviewer: ```suggestion
Creating a zone is similar to creating an API Client, you need to specify the customer's account when creating the zone.
```

Allowed rate plans are:

* PARTNERS_PRO
* PARTNERS_BIZ
* PARTNERS_ENT

> These are the same plans that customers buy in the dashboard. Additionally, depending on your agreement there may be additional services you can select including **dedicated_certificates**, **dedicated_certificates_custom**, **page_rules**, etc. These are specified as `component_values` in the subscriptions creation request.

Reviewer: I'd actually leave these as code values b/c they're params you're passing in during an api call.
